lib/Doctrine/ORM/Mapping/Driver/XmlDriver.php

```diff
@@ -255,11 +255,20 @@ public function loadMetadataForClass($className, ClassMetadata $metadata)
 
         if (isset($xmlRoot->embedded)) {
             foreach ($xmlRoot->embedded as $embeddedMapping) {
+                $columnPrefix = isset($embeddedMapping['column-prefix'])
```

Comment:
I've said before, I'm not making use of a additional attribute as you've proposed.

I'd really think that there's no need for adding another attribute, well, we only need ensure if `false` or `0` was given for `column-prefix`.

It will be always string, of course, we can't change it, but, it's only input treatment for us, if false was given, so we understand the `false` as that column should not be prefixed. The only difference is that any incoming value from a XML attribute will be always string :)